backend/db/db.go

```diff
@@ -0,0 +1,5 @@
+package db
+
+import "gorm.io/gorm"
+
+var DB *gorm.DB
```

Comment:
We need to access DB in user model so leaving this in config makes a circular dependency.

Reply:
Is this like a webauthn thing, I dont see the circular dependency ngl

Reply:
The user model requires the database, so leaving this in config would result in user model importing config and config importing user model